### ISSUE TITLE
[Feature Request] Implement noise suppression

### ISSUE THREAD

**Issue by ndren** [qTox/qTox#6231]
I think software-level noise suppression could help significantly when there is no hardware-level support. Perhaps an opt-in in the settings. I'm not sure how this could be implemented, but I think it should be within the scope as e.g. screen sharing is a feature.

**Comment by vicfic18** [comment]
There is significant echo in some video calls. The background noise makes it worse.
.

**Comment by shinodem** [comment]
Something like rtx voice =)

**Comment by anthonybilinski** [comment]
I agree this is a needed feature, but this issue is a duplicate of https://github.com/qTox/qTox/issues/1655 so I will close it. It's still not implemented due to us needing a cross platform echo cancellation library, since it would be very difficult to implement ourselves.

**Comment by anthonybilinski** [comment]
Sorry reading again this is probably a different feature and generally easier to implement. I'll re-open and leave noise suppression as a separate request from echo cancellation.
